fastlane/swift/Actions.swift

```diff
@@ -1,16 +1 @@
-// Actions.swift
-// Copyright (c) 2022 FastlaneTools
-
-// This autogenerated file will be overwritten or replaced when running "fastlane generate_swift"
-//
-//  ** NOTE **
-//  This file is provided by fastlane and WILL be overwritten in future updates
-//  If you want to add extra functionality to this project, create a new file in a
-//  new group so that it won't be marked for upgrade
-//
-
 import Foundation
-
-// Please don't remove the lines below
```

Comment:
I think we don't need to change this file

Reply:
it's changed by the `Fastlane` automatically after adding the plugin. These changes are all `remove`, so I think it's fine